doc-content/enterprise-only/openshift/rhdm711-authoring-ha-ref.adoc

```diff
@@ -75,7 +75,7 @@ https://access.redhat.com/documentation/en-us/openshift_container_platform/3.11/
 |`DECISION_CENTRAL_MEMORY_REQUEST` | -- | Decision Central Container memory request. | 3Gi | True
 |`DECISION_CENTRAL_CPU_REQUEST` | -- | Decision Central Container CPU request. | 1500m | True
 |`KIE_SERVER_MEMORY_LIMIT` | -- | KIE Server Container memory limit. | 2Gi | True
-|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1.5Gi | True
+|`KIE_SERVER_MEMORY_REQUEST` | -- | KIE Server Container memory Request. | 1536Mi | True
```

Comment:
[technical] Most size values are stated in `Gi`, is there a reason for changing this to `1536Mi`?